snprc_ehr/resources/queries/snprc_ehr/ExportSndLookups.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
+    LookupId as LOOKUP_ID,
+    ls.SetName as LOOKUP_KEY,
+    Value as VALUE,
+    CASE
+        WHEN l.Displayable = 0 THEN 'Y'
+        WHEN l.Displayable = 1 THEN 'N'
+    END as IS_HIDDEN,
+    l.SortOrder as ORDER_NUM,
+    l.ObjectId as OBJECT_ID,
+    'N' as DEFAULT_FLAG,
```

Comment:
Can I please know the purpose of this Default flag always 'N' ?

Reply:
There is no default flag field in the SND table, so Terry decided CAMP should not have the default flag set anymore when the data is pulled from the ETL, as discussed yesterday in our Labkey overview meeting. The user will now have to set the default value manually in CAMP

Reply:
Thank You Jay for clarifying.